src/main/java/suneido/runtime/builtin/Lucene.java

```diff
@@ -153,9 +158,9 @@ void close() {
 
 	static IndexWriter writer(String path, boolean create) {
 		try {
-			Directory dir = FSDirectory.open(new File(path));
+			Directory dir = FSDirectory.open(getPath(path));
```

Comment:
This getPath could just be Paths.get since path is already a string

Reply:
It could be. I simply did it to standardize the calling of Paths. Even though it is a string it is handled the same